### DISCUSSION TITLE
Problems with cingular pole projection on white matter mesh

### DISCUSSION THREAD

**Discussion by AnnaBhlr** [brainvisa/brainvisa.github.io#158] [General]
Hi,

I got the process running to find estimates of the cingular wall in my white matter mesh. That's the command I used:

`axon-runprocess CingularPoleProjection white_mesh=/path/tosubject/t1mri/default_acquisition/default_analysis/segmentation/mesh/subjectnumber_Lwhite.gii side='left' subject_transformation=/path/tosubject/t1mri/default_acquisition/registration/RawT1-subjectnumber_default_acquisition_TO_Talairach-ACPC.trm pole_template=/home/brainvisa/build/bug_fix/share/brainvisa-share-4.6/hemitemplate/Template_Cingular.nii.gz template_pole_transformation=/home/brainvisa/build/bug_fix/share/brainvisa-share-4.6/hemitemplate/RawT1-Template_TO_Talairach-ACPC.trm pole=/data/testpoletexture.gii"`

I was not possible to just run it with my white_mesh as input. I additional gave the subject specific transformation in and I searched throughout BrainVisa and took what seem to be the cingular pole template and transformation. But the resulting projected cingular pole looks out of place. What went wrong here (blue dots correspond to the results from the process)? Does the mesh need to be flipped before?

<img width="368" alt="Bildschirm­foto 2023-11-17 um 11 30 55" src="https://github.com/brainvisa/brainvisa.github.io/assets/131627125/d493db69-bad7-4af7-ab9e-7e26c982fdff">

**Comment by denisri** [comment]
Parameters don't look wrong to me. Maybe you can play with the dilation and erosion parameters to see if you end up with a better result. Bus as I said earlier, this process has not been designed to precisely segment the cingular pole, but rather to get a center point, so maybe it doesn't fits your needs.
I don't know if people from Marseille MecaLab (who wrote this process) are reading this and can comment on it or not...

**Reply by AnnaBhlr** [reply]
Thanks for your answer! I can see your point re the process is designed to get a center point. I'd like to use the meshes (approx 3000 participants) for further analysis. In order to get this further analysis running I need to cut out the cingular pole. So the estimation of the cingular pole does not need to be 100% perfect. Is there any way to do this using BrainVisa? I know that I could just re-preprocess the T1w scans using Freesurfer for labeled meshes but then I would loose some important information I only get from the Morphologist pipeline (which I rather would not).
